modules/testserver/src/test/java/com/smartbear/readyapi4j/testserver/execution/TestServerRequestExecutorTest.java

```diff
@@ -71,9 +72,19 @@ public void sendsNotificationsOnAsynchronousRequests() throws Exception {
 
         recipeExecutor.addExecutionListener(executionListener);
         recipeExecutor.submitRecipe(recipeToSubmit);
+        verify(executionListener).executionStarted(argThat(new ArgumentMatcher<Execution>(){
```

Comment:
At some point we should make all the anonymous ArgumentMatcher classes into lambdas. Much more readable. Not a merge-blocker though.